packages/toolpad-components/src/Text.tsx

```diff
@@ -23,9 +22,22 @@ interface TextProps extends Omit<BaseProps, 'children'> {
 
 const MarkdownContainer = styled('div')({
   display: 'block',
-  [`&:empty::before`]: { content: '""', display: 'inline-block' },
+  maxWidth: 'fill-available',
+  '& span:empty::before': {
```

Comment:
So that the min-height triggers when the lazy import is resolving?

```suggestion
  '&:empty::before, & span:empty::before': {
```

Reply:
Also, we don't want to trigger `<span>` that are rendered by the markdown, so only the first direct children

```suggestion
  '&:empty::before, & > span:empty::before': {
```